src/auto-evo/simulation/PopulationSimulation.cs

```diff
@@ -18,7 +18,7 @@ public static void Simulate(SimulationConfiguration parameters)
 
             while (parameters.StepsLeft > 0)
             {
-                RunSimulationStep(parameters, speciesToSimulate, random);
```

Comment:
I would like to keep the random as a parameter as I can imagine many scenarios where the simulation step needs randomness even if it isn't currently needed,
add `_ = random` to stop it from being detected as unused.